From: Jiang Xin <worldhello.net@gmail.com>
Subject: [PATCH] No stop to edit for the new merge behavior of git

In Git 1.7.10, Merge will stop and wait for a merge commit log. This
backward-incompatible improvement that will break topgit. To fix it,
just export GIT_MERGE_AUTOEDIT=no.

See: http://git-blame.blogspot.jp/2012/02/anticipating-git-1710.html

Signed-off-by: Jiang Xin <worldhello.net@gmail.com>

---
 tg.sh |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/tg.sh b/tg.sh
index 9082d88..b7661c2 100644
--- a/tg.sh
+++ b/tg.sh
@@ -430,6 +430,8 @@ get_temp()
 ## Initial setup
 
 set -e
+# suppress the merge log editor feature since git 1.7.10
+export GIT_MERGE_AUTOEDIT=no
 git_dir="$(git rev-parse --git-dir)"
 root_dir="$(git rev-parse --show-cdup)"; root_dir="${root_dir:-.}"
 # Make sure root_dir doesn't end with a trailing slash.
-- 
tg: (d279e29..) t/git-merge-no-edit (depends on: tgmaster)
